lib/identifiers/pubmed_id.rb

```diff
@@ -1,7 +1,7 @@
 module Identifiers
   class PubmedId
     ZERO_PADDED_NUMBER = %r{(?<=^|[[:space:]])0*(?!0)(\d+)(?=$|[[:space:]])}
-    PUBMED_URL = %r{https?://(?:www\.)?ncbi\.nlm\.nih\.gov/(?:m/)?pubmed/0*(\d+)}i
+    PUBMED_URL = %r{(?:ncbi\.nlm\.nih\.gov/(?:m/)?pubmed/|pmid:|info:pmid/)0*(\d+)}i
```

Comment:
Should we retain the `https?` and `www` part of the pattern here? We had to drop it in the PHP version because it's part of a lookbehind and you can only use a restrictive subset of regex features in those.